src/saveToCase.js

```diff
@@ -568,6 +687,25 @@ define([
                         return;
                     }
                 }
+                
+                var attachmentRegex = /\/case\/attachment\/(\w+)\/@src/,
+                    attachRet = path.match(attachmentRegex);
+                if (attachRet) {
+                    basePath = path.replace(attachmentRegex, "");
+                    mug = form.getMugByPath(basePath);
+                    if (mug && mug.__className === "SaveToCase") {
+                        var attachProperties = mug.p.attachment_property,
+                            nodeName = attachRet[1];
+                        if (!attachProperties) {
+                            attachProperties = {};
```

Comment:
Hmm. I'm not sure this is actually a necessary check here. The data tree should have already parsed and initialized `attachmentProperty`. If it isn't in the data tree then a warning should pop up that there was a bind node without a data component right?